springerlink.md

```diff
@@ -1,163 +1,1552 @@
+</td>
+</tr>
+<tr id="three-flashes-or-below-threshold" valign="top">
+<td><a href="http://www.w3.org/TR/WCAG20/#seizure-does-not-violate"><strong>2.3.1 Three Flashes or Below Threshold</strong></a> (Level A)
+<br><br>Also applies to:<br><br>
+<em>EN 301 549 Criteria</em>
+<ul>
+<li>9.2.19 (Web)</li>
+<li>10.2.19 (non-web document)</li>
+<li>11.2.1.19 (Software)</li>
+<li>11.2.2.19 (Closed Software) – Does not apply</li>
+<li>11.6.2 (Authoring Tool)</li>
+<li>12.1.2 (Product Docs)</li>
+<li>12.2.4 (Support Docs)</li>
+</ul>
+<em>Revised Section 508</em>
+<ul>
+<li>501 (Web)(Software)</li>
+<li>504.2 (Authoring Tool)</li>
+<li>602.3 (Support Docs)</li>
+</ul></td>
+<td>Supports</td>
+<td>The pages of this product do not contain anything that flashes.</td>
+</tr>
+<tr id="bypass-blocks" valign="top">
+<td><a href="http://www.w3.org/TR/WCAG20/#navigation-mechanisms-skip"><strong>2.4.1 Bypass Blocks</strong></a> (Level A)
+<br><br>Also applies to:<br><br>
+<em>EN 301 549 Criteria</em>
+<ul>
+<li>9.2.20 (Web)</li>
+<li>10.2.20 (non-web document) – Does not apply</li>
+<li>11.2.1.20 (Software) – Does not apply</li>
+<li>11.2.2.20 (Closed Software) – Does not apply</li>
+<li>11.6.2 (Authoring Tool)</li>
+<li>12.1.2 (Product Docs)</li>
+<li>12.2.4 (Support Docs)</li>
+</ul>
+<em>Revised Section 508</em>
+<ul>
+<li>501 (Web)(Software) – Does not apply to non-web software</li>
+<li>504.2 (Authoring Tool)</li>
+<li>602.3 (Support Docs) – Does not apply to non-web docs</li>
+</ul></td>
+<td>Partially Supports</td>
+<td>
+The pages found at link.springer.com generally use semantically-correct markup so that Assistive Technology users can recognise elements and move around them accordingly. We make use of ARIA roles to support landmarks. We also provide skip links for bypassing repetitive navigation elements.
+<br><br>
+Skip links are not present on pages located at checkout.springer.com. Landmarks are present but malformed, and may be difficult to use. 
+<br><br>
+The OneTrust "manage cookies" privacy centre (third party content) does not provide a way for screen reader users to bypass the links in its left hand menu.
+</td>
+</tr>
+<tr id="page-titled" valign="top">
+<td><a href="http://www.w3.org/TR/WCAG20/#navigation-mechanisms-title"><strong>2.4.2 Page Titled</strong></a> (Level A)
+<br><br>Also applies to:<br><br>
+<em>EN 301 549 Criteria</em>
+<ul>
+<li>9.2.21 (Web)</li>
+<li>10.2.21 (non-web document)</li>
+<li>11.2.1.21 (Software) - Does not apply</li>
+<li>11.2.2.21 (Closed Software) – Does not apply</li>
+<li>11.6.2 (Authoring Tool)</li>
+<li>12.1.2 (Product Docs)</li>
+<li>12.2.4 (Support Docs)</li>
+</ul>
+<em>Revised Section 508</em>
+<ul>
+<li>501 (Web)(Software)</li>
+<li>504.2 (Authoring Tool)</li>
+<li>602.3 (Support Docs)</li>
+</ul></td>
+<td>Supports</td>
+<td>SpringerLink pages have unique page titles that describe the purpose of the page.</td>
```

Comment:
Not all pages have unique titles. There are articles that get updated and the older versions are still available. So unless the title changes in the new revision, there is no distinction between the page titles.

Reply:
Ooh, good catch. Do you know of an example you can show me?

Reply:
Yup, here is a living review article that has a revision:

https://link.springer.com/article/10.12942/lrsp-2004-1 - original
https://link.springer.com/article/10.12942/lrsp-2009-4 - latest

You can tell whether an article has a history by the `View article history` link below author(s) and it will anchor you down to a section that lists all the available versions.